product-development/README.md

```diff
@@ -33,6 +33,35 @@ Anyone advocating for an epic's priority (or priorities in general) should @-men
 
 <img src="images/epics-priority.png" height="100" />
 
+### Estimating Stories
+
+Right now we have Clubhouse configured to use the following, roughly Fibonacci sequence story point estimate scale:
+
+* 1
+* 2
+* 3
+* 5
+* 8
+* 13
+* 21
+
+We use story points for one purpose: to communicate with each other about the complexity of work. As an engineer this is useful when it's 4 PM and I'm looking at an epic I'm working on it's helpful to know if a story is _expected_ to be simple or complicated. As a PM/EM/CEO this is useful when I want to ask questions like "roughly how much work is left on X" and "when should we start work on marketing for Y".
```

Comment:
```suggestion
We use story points for one purpose: to communicate with each other about the complexity of work. As an engineer this is useful when it's 4 PM and I'm looking at an epic I'm working on it's helpful to know if a story is _expected_ to be simple or complicated.
```

@jazzdan : We don't have EMs or PMs (at least in title) just yet. So let's not write down something that we don't need right now. (I think it's great to start thinking about these things, but perhaps not in this section.)

I think in the first iteration of formalizing (or just documenting) a point system, we should _not_ use it to communicate expectations of when an epic will be done that informs marketing, as in your example here. Since that would require non-engineers to start adding points and assuming that all points contribute to the entire epic. And that requires work on both engineers and non-engineers for clear communications. Rather, if we wanted to communicate when things will be done, we can just use end dates, which we have a few times since I've joined. Points have a lot of other uses, I just rather we not apply them directly to estimating due dates, yet.